### DISCUSSION TITLE
Does SeisSol support PML at present ?

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#805] [Q&A] [answered]
I see PML is still under construcyion, and support CPML(Convolutional Perfectly Matched Layer) ?
![image](https://user-images.githubusercontent.com/104721358/221506556-e1a617e2-26f1-47a5-8e3d-4ce97f3477e9.png)

![image](https://user-images.githubusercontent.com/104721358/221506200-05392df5-2812-4a71-9845-cf7855edbeb0.png)

I notice that WP2_LOH1 doesn't set the SpongeLayer, and `SpongeLayer NAMELIST` include `PMLDelta`,`PMLFrequency` and so on, if I want set CPML,how can I set it ?
![image](https://user-images.githubusercontent.com/104721358/221551903-fe2e0d7f-628c-49ef-a048-3ad82f6b40f7.png)

![image](https://user-images.githubusercontent.com/104721358/221507528-23c4064d-f2a6-419d-b56f-1f0be7b25d8e.png)

**Comment by Thomas-Ulrich** [comment, marked as answer]
no, SeisSol does not have and never had CPML fully implemented, unfortunately.

**Reply by smallriver666** [reply]
> no, SeisSol does not have and never had CPML fully implemented, unfortunately.

 Does SeisSol  support PML ?

**Reply by krenzland** [reply]
Only features that are mentioned in the official documentation (read the docs) are supported.
